src/components/media-player/media-player.component.ts

```diff
@@ -35,6 +35,9 @@ export class MediaPlayerComponent implements AfterViewInit, OnDestroy {
     /** The `anonymous` keyword means that there will be no exchange of user credentials when the media source is fetched. */
     @Input() crossorigin: 'use-credentials' | 'anonymous' = 'use-credentials';
 
+    @Input()
```

Comment:
Can we add a comment above the input describing the purpose. This allows editors to show documentation about the input without requiring the user to view the docs site.